package.json

```diff
@@ -9,13 +9,14 @@
     "start:api": "node ./dist/api/server.js",
     "setup": "yarn setup:api && yarn setup:demo",
     "setup:api": "cross-env NODE_PATH=. rm -rf ./dist/api && tsc -p ./api --outDir dist/api",
-    "setup:demo": "yarn build && yarn copy:files",
+    "setup:demo": "yarn build && yarn copy:files && yarn copy:demo-assets",
     "build": "yarn build:lib && yarn build:demo",
     "build:lib": "cd projects/media-viewer && yarn install && ng build --prod",
     "build:demo": "ng build media-viewer-demo-app --prod",
     "copy:files": "yarn copy:js-dependencies && yarn copy:assets && yarn copy:docs",
     "copy:js-dependencies": "cross-env NODE_PATH=. rsync -r projects/media-viewer/node_modules/pdfjs-dist/build dist/media-viewer/assets",
     "copy:assets": "cross-env NODE_PATH=. rsync -r projects/media-viewer/src/assets/* dist/media-viewer/assets/",
+    "copy:demo-assets": "cross-env NODE_PATH=. rsync -r dist/media-viewer/assets/* dist/demo-app/assets/",
```

Comment:
could we restructure the copy:assets tasks into copy:assets-lib, copy:assets-demo
also, maybe copy it directly from source so that we're not relying on the other task

Reply:
yes, make sense, will do that